crypto/fipsmodule/evp/evp.c

```diff
@@ -290,7 +290,7 @@ EVP_PKEY *EVP_PKEY_new_mac_key(int type, ENGINE *engine, const uint8_t *mac_key,
   if(key == NULL) {
     goto err;
   }
-  key->key = mac_key;
+  key->key = OPENSSL_memdup(mac_key, mac_key_len);
```

Comment:
we're allocating here; do we need to handle failure?

Reply:
yes